### PR TITLE
[MS Graph API] Expand API support

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/42457

## Description
Added support for `auditLogs/signIns` in **msgraph-api-request**.

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py [hunk 1]
```diff
@@ -106,12 +106,20 @@ def generic_command(client: MsGraphClient, args: Dict[str, Any]) -> CommandResul
...
 
         if argToBoolean(args.get('populate_context', 'true')):
-            results['outputs'] = response.get('value')
+            results['outputs'] = get_response_outputs(response)
             results['outputs_prefix'] = 'MicrosoftGraph'
 
     return CommandResults(**results)  # type: ignore[arg-type]
 
 
+def get_response_outputs(response: dict):
```
Comment (Itay4) [inline Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py:115]: please add return type hint

File: Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py [hunk 2]
```diff
@@ -106,12 +106,20 @@ def generic_command(client: MsGraphClient, args: Dict[str, Any]) -> CommandResul
...
+    if 'value' in response:
+        return response['value']
```
Comment (Itay4) [inline Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py:117]: dont we need to pop out `@odata.context` here as well ?

File: Packs/MicrosoftGraphAPI/ReleaseNotes/1_0_7.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Microsoft Graph API
+- Added support in ***msgraph-api-request*** for API endpoints that don't return their results under **value** key.
```
Comment (Itay4) [inline Packs/MicrosoftGraphAPI/ReleaseNotes/1_0_7.md:3]:
```suggestion
- Added support in ***msgraph-api-request*** for API endpoints that don't return the results under the **value** key.
```

**Review by Itay4 — APPROVED** [review]

File: Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py (hunk 2 above)
Comment (DeanArbel) [reply Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py:117]: no, since it's in the root, but not in the `value`.

**Pull request merged**